hybridse/src/udf/literal_traits.h

```diff
@@ -408,26 +408,50 @@ struct DataTypeTrait<Tuple<T...>> {
     }
 };
 
+// DataTypeTrait::CCallArgType
 template <typename T>
 struct CCallDataTypeTrait {
     using LiteralTag = T;
+
+    // Get the distinct(from value and type) byte representation of the input
+    template <typename TT = T, std::enable_if_t<std::is_integral_v<TT> || std::is_floating_point_v<TT>, int> = 0>
+    static absl::string_view to_bytes_ref(TT* data) {
+        const auto* bytes = reinterpret_cast<const char*>(data);
+        return absl::string_view(bytes, sizeof(TT));
+    }
 };
+
 template <typename V>
 struct CCallDataTypeTrait<V*> {
     using LiteralTag = Opaque<V>;
 };
 template <>
 struct CCallDataTypeTrait<openmldb::base::Timestamp*> {
     using LiteralTag = openmldb::base::Timestamp;
+
+    static absl::string_view to_bytes_ref(openmldb::base::Timestamp** data) {
+        auto& ts = (*data)->ts_;
+        return CCallDataTypeTrait<decltype(openmldb::base::Timestamp::ts_)>::to_bytes_ref(&ts);
```

Comment:
this may produce exact same result to the same int64 input of `ts_`, the same apply to `Date` below. Discussion may necessary.